src/main/java/mate/jdbc/dao/ManufacturerDaoImpl.java

```diff
@@ -0,0 +1,119 @@
+            createRecordStatement.setString(2, manufacturer.getCountry());
+            createRecordStatement.executeUpdate();
+            ResultSet generatedKeys = createRecordStatement.getGeneratedKeys();
+            if (generatedKeys.next()) {
+                manufacturer.setId(generatedKeys.getObject(1, Long.class));
+            }
+        } catch (SQLException e) {
+            throw new RuntimeException("Can`t insert record to manufacturers table", e);
+        }
+        return manufacturer;
+    }
+
+    @Override
+    public Optional<Manufacturer> get(Long id) {
+        Optional<Manufacturer> result;
+        String getForIdString = "SELECT * FROM manufacturers WHERE id = ? AND is_deleted = FALSE";
+        try (Connection connection = connectToDB.getConnection();
+                PreparedStatement getForIdStatement =
+                        connection.prepareStatement(getForIdString)) {
+            getForIdStatement.setString(1, id.toString());
+            ResultSet resultSet = getForIdStatement.executeQuery();
+            result = resultSet.next()
+                    ? Optional.of(getInstanceManufacturer(resultSet)) : Optional.empty();
+        } catch (SQLException e) {
+            throw new RuntimeException("Can`t get record by id from DB", e);
+        }
+        return result;
+    }
+
+    @Override
+    public List<Manufacturer> getAll() {
+        List<Manufacturer> manufacturers;
+        String getAllString = "SELECT * FROM manufacturers WHERE is_deleted = FALSE";
+        try (Connection connection = connectToDB.getConnection();
+                PreparedStatement getAllStatement =
+                        connection.prepareStatement(getAllString)) {
+            ResultSet resultSet = getAllStatement.executeQuery();
+            manufacturers = new ArrayList<>();
+            while (resultSet.next()) {
+                manufacturers.add(getInstanceManufacturer(resultSet));
+            }
+        } catch (SQLException e) {
+            throw new RuntimeException("Can`t get records from DB", e);
+        }
+        return manufacturers;
+    }
+
+    @Override
+    public Manufacturer update(Manufacturer manufacturer) {
+        String updateRecordString = "UPDATE manufacturers SET name = ?, country = ? WHERE id = ?";
+        try (Connection connection = connectToDB.getConnection();
+                PreparedStatement updateRecordStatement =
+                        connection.prepareStatement(updateRecordString)) {
+            updateRecordStatement.setString(1, manufacturer.getName());
+            updateRecordStatement.setString(2, manufacturer.getCountry());
+            updateRecordStatement.setString(3, manufacturer.getId().toString());
+            updateRecordStatement.executeUpdate();
+        } catch (SQLException e) {
+            throw new RuntimeException("Can`t update record", e);
+        }
+        return get(manufacturer.getId()).orElse(null);
```

Comment:
```suggestion
        return manufacturer;
```

It's better to return the same object, null definitely doesn't fit there